coretex/cli/modules/utils.py

```diff
@@ -22,6 +22,22 @@
 
 import click
 
+from ...utils.process import command
+
+
+def getVersion() -> Optional[str]:
+    _, output, _ = command(["pip", "show", "coretex"], ignoreStdout = True, ignoreStderr = True)
```

Comment:
There is a very simple way to do this without parsing command outputs. Let's use that instead of this complex approach.